torchci/README.md

```diff
@@ -87,10 +87,21 @@ console.
 4. Update `rockset/prodVersion.json` with the new version of the lambda.
 
 
+### Alerts
+
+The scripts/check_alerts.py queries HUD, filters out pending jobs, and then checks to see if there are 2 consecutive
+SHAs that have the same failing job. If it does, it will either create a new Github Issue or update the existing
+Github Issue.
+
+Once the issue is created, internal webhooks creates a task out of the issue. Then a [butterfly bot
+rule](https://www.internalfb.com/butterfly/rule/2024866984357962) will trigger to assign the task to 
```

Comment:
Just curious: is there any policy preventing us putting internal links here (or in other OSS repos for that matter)?

Reply:
No it's fine to put internal links here. You can't access internal unless you're on VPN/certain certs so it should be fine. External people will just have to depend on us to edit butterfly bot stuff.